components/scream/src/share/error_defs.hpp

```diff
@@ -1,6 +1,8 @@
 #ifndef SCREAM_ERROR_DEFS_HPP
 #define SCREAM_ERROR_DEFS_HPP
 
+#include <string>
```

Comment:
Sooner rather than later we probably want assertion and throw macros that take a comm, so the output can include the rank. Doesn't have to be this PR, though.

Reply:
Yes, at some point we should have a PR focused on debug/assert/exceptions.